### DISCUSSION TITLE
Unable to Generate Employee Login Credentials on DIGIT Sandbox - Receiving 404 Error

### DISCUSSION THREAD

**Discussion by arun-ganachari** [egovernments/Digit-Core#609] [Sandbox]
I was attempting to generate employee login credentials on the DIGIT eGov sandbox platform to explore employee functionalities. However, when I clicked on the provided link, I encountered a 404 error. Attached screenshots for reference.

I was following below link where it shows how to generate the employee login.
https://urban.digit.org/ecosystem/access-digit

![image](https://github.com/user-attachments/assets/1b9dd40c-869e-4fde-96c0-8aa438959427)
![image](https://github.com/user-attachments/assets/3b6b5e19-69be-44c4-9895-f5901696737d)

Has anyone encountered this issue before? Is there an alternate way to generate employee login credentials, or could there be a configuration issue on the sandbox environment?

**Comment by Rishabh-egov** [comment]
Hi @arun-ganachari 
Thanks for reporting this to us , the error is coming up since we have migrated to a new website which doesn't have the page. We will get this fixed soon.

**Comment by Rishabh-egov** [comment]
Hi @arun-ganachari we have fixed it , please check if you are able to access it now ?
